base/REPLCompletions.jl

```diff
@@ -130,8 +130,12 @@ function complete_path(path::AbstractString, pos)
             push!(matches, id ? file * (@windows? "\\\\" : "/") : file)
         end
     end
-    matches = UTF8String[replace(s, r"\s", "\\ ") for s in matches]
-    return matches, nextind(path, pos - sizeof(prefix) - length(matchall(r" ", prefix))):pos, length(matches) > 0
+    matches = UTF8String[replace(s, r" ", "\\ ") for s in matches]
```

Comment:
Ok. The only case the code take into account since it replaces `"\s"` with `"\\ "` but it should then be replaced like: `replace(s, r"(\s)", "\\1")`? I will revert this line change and then correct it in another pull because there are multiple other places that then needs to be changed.